Ecommerce/index.html

```diff
@@ -1,13 +1,32 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>E-commerce</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for ECOMMERCE TASK -->
-    <!-- to import image from public folder, do it like this = <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+
+<!DOCTYPE html>
+<html><head><title>Task1</title></head>
+<body>
+<h1>Welcome to My Shop</h1>
+<img src="shp.png">
+<ul>
+<li><a href="#">About Us</a></li>
+<li><a href="#">Products</a></li>
+<li><a href="#">Contact Us</a></li>
+</ul>
+<h2>About Us</h2>
+Welcome to My Shop! We are a one -stop destination for all your shopping needs. Our mission is to provide you with the best products and exceptional customer service.
+<h2>Products</h2>
+Explore our wide range of products, including:
+<ul>

```

Comment:
The button element should have a type attribute specified, such as 'submit' or 'button'.